package.json

```diff
@@ -30,24 +30,28 @@
     "./package.json": "./package.json"
   },
   "dependencies": {
-    "lodash-es": "^4.17.21"
+    "lodash-es": "^4.17.21",
+    "@ckeditor/ckeditor5-integrations-common": ">=1.0.0 || ^0.0.0-nightly"
   },
   "peerDependencies": {
     "ckeditor5": ">=42.0.0 || ^0.0.0-nightly",
+    "ckeditor5-premium-features": ">=42.0.0 || ^0.0.0-nightly",
     "vue": "^3.4.0"
   },
   "devDependencies": {
     "@ckeditor/ckeditor5-dev-bump-year": "^40.1.0",
     "@ckeditor/ckeditor5-dev-ci": "^40.1.0",
     "@ckeditor/ckeditor5-dev-release-tools": "^40.1.0",
     "@ckeditor/ckeditor5-dev-utils": "^40.1.0",
+    "@ckeditor/ckeditor5-integrations-common": "file:../ckeditor5-integrations-common",
```

Comment:
Maybe it would be better to add a description to the PR of how to test it rather than hardcoding local dependency in `package.json`?

For me the first attempt to test this PR ended with an error at the very beginning when installing dependencies. To test this PR I had to discover that it requires `ckeditor/ckeditor5-integrations-common` cloned next to the `ckeditor/ckeditor5-vue` on a `ck/cdn-integration` branch, and additionally it must have `dist` built.

Reply:
Yep, sorry. We're actively working on merging and releasing `0.0.1` and it'll be updated as soon as we do that.